libraries/AP_WindVane/AP_WindVane.cpp

```diff
@@ -49,8 +50,8 @@ const AP_Param::GroupInfo AP_WindVane::var_info[] = {
 
     // @Param: DIR_PIN
     // @DisplayName: Wind vane analog voltage pin for direction
-    // @Description: Analog input pin to read as wind vane direction
-    // @Values: 11:Pixracer,13:Pixhawk ADC4,14:Pixhawk ADC3,15:Pixhawk ADC6/Pixhawk2 ADC,50:AUX1,51:AUX2,52:AUX3,53:AUX4,54:AUX5,55:AUX6,103:Pixhawk SBUS
+    // @Description: Analog input pin to read as wind vane direction. If the Wind Vane type is Angle Sensor, this parameter selects the Angle Sensor Instance.
```

Comment:
I think my preference would be to encode this in the type. Re-using pin is a bit odd. Maybe we would skip some and start at 20 for type. 

"20:AngleSensor 1, 21:AngleSensor 2, 22:AngleSensor 3" ect.

Or I guess we flip it the other way from the start and have a function in the angle sensor lib. We usually endup doing that anyway when there are several instances. Servos and RC are examples, and we have just done relay.

Reply:
I think I'm going to go with the first option - that's how it's done for RPM which is the closest example I can think of. I think that the function flip makes the most sense for outputs which can only be assigned one function at a time. In theory, an angle sensor could be consumed by multiple devices at a time.